plugin/buycourses/src/course_panel.php

```diff
@@ -39,23 +44,17 @@
     }
 }
 
-$toolbar = Display::toolbarButton(
-    $plugin->get_lang('CourseListOnSale'),
-    'course_catalog.php',
-    'search-plus',
-    'primary',
-    ['title' => $plugin->get_lang('CourseListOnSale')]
-);
+$interbreadcrumb[] = ['url' => '../index.php', 'name' => $plugin->get_lang('UserPanel')];
```

Comment:
i will create another support task for this, maybe for not use a breadcrumb ... maybe a plugin menú?